fe/src/components/BurgerMenu/BurgerMenu.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+import classNames from 'classnames';
+
+import heartLike from '../../assets/images/icons/favourite.svg';
+import logo from '../../assets/images/icons/logo.svg';
+import cross from '../../assets/images/icons/close.svg';
+import bag from '../../assets/images/icons/bag.svg';
+
+import { Link, NavLink } from 'react-router-dom';
+
+import './burger-menu.scss';
+
+export const BurgerMenu: React.FC = () => {
+  const [showBurgerMenu, setShowBurgerMenu] = useState(false);
+
+  const changeBurgerState = () => {
+    setShowBurgerMenu(true);
+  };
+
+  return (
+    <header
+      className={classNames('burger-menu', {
+        'hide-burger-menu': showBurgerMenu,
```

Comment:
```suggestion
        'hide-burger-menu': hideBurgerMenu,
```
Doesn't look logical. Could you change the name of the variable or class so they fit each other